recipes/hupper/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "hupper" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb3778398658a011c96e620adcd73175f306f880a6d86b2ebb6d2a15a74b6b9b
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - hupper = hupper.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

Comment:
The [new recommended pip command](https://conda-forge.org/docs/meta.html#use-pip) is `"{{ PYTHON }} -m pip install . -vv"`.

Reply:
:+1:

Reply:
Done